lib/kubernetes-deploy/kubernetes_resource/deployment.rb

```diff
@@ -74,7 +81,7 @@ def pretty_timeout_type
 
     def deploy_timed_out?
       # Do not use the hard timeout if progress deadline is set
-      @progress_condition.present? ? deploy_failing_to_progress? : super
+      super #@progress_condition.present? ? deploy_failing_to_progress? : super
```

Comment:
Are we assuming that deployments always use progressDeadlineSeconds and never the timeout in this file?